app/models/git_repository.rb

```diff
@@ -24,9 +24,8 @@ def checkout_workspace(work_dir, git_reference)
     executor.output.write("# Beginning git repo setup\n")
 
     ensure_mirror_current &&
-    create_workspace(work_dir) &&
-    checkout(git_reference, work_dir) &&
-    checkout_submodules(work_dir)
+      checkout(git_reference, work_dir) &&
```

Comment:
Could put all of these on one line?

Reply:
would get kinda crowded and the alternative is `return unless foo; return unless bar` 🤷‍♂️